ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+<!--
+Thank you for reporting an issue! For feature requests, scroll down.
+
+###########################
+Regarding problem reports:
+###########################
+
+Before you submit a problem report, please do the following:
+
+  * Search the list of existing issues to see if someone has reported the issue already.
+  * Update to the latest master branch of SeqAn and see if the issue persists.
+
+If you already know how to fix the issue, you can directly create a pull-request instead of an issue. See the guide on [CONTRIBUTING](./CONTRIBUTING.md) for more details.
+-->
+
+## Platform
+
+  - SeqAn version: <!-- enter version here -->
+  - Operating system: <!-- enter output of `uname -a` here-->
+  - Compiler: <!-- enter output of `/path/to/compiler --version` here -->
+
+## Description
+
+### How to repeat the problem
+
+<!-- step to repeat the problem, code snippets... -->
```

Comment:
```suggestion
<!-- step to repeat the problem, code snippets, compiler flags... -->
```
In general I think that steps to reproduce the error is already including how the code was compiled but maybe writing it explicitly makes it more clear?